src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -16,17 +17,24 @@ const chatMessageStyles: ComponentSlotStylesInput<ChatMessageProps, ChatMessageV
     maxWidth: v.messageWidth,
     wordBreak: 'break-word',
     wordWrap: 'break-word',
+    ':focus': {
+      outline: 'none',
+      '& .ui-chat__message__messageBody': {
+        outline: `.2rem solid ${siteVariables.brand}`,
+      },
+    },
   }),
 
   avatar: ({ props: p }: { props: ChatMessageProps }): ICSSInJSStyle => ({
+    flex: 'none',
     display: p.mine ? 'none' : undefined,
     marginTop: px10asRem,
     marginBottom: px10asRem,
     marginLeft: p.mine ? px10asRem : 0,
     marginRight: p.mine ? 0 : px10asRem,
   }),
 
-  content: ({ props: p, variables: v }): ICSSInJSStyle => ({
+  messageBody: ({ props: p, variables: v }): ICSSInJSStyle => ({
```

Comment:
when we were using `Layout`, we had `display: 'grid'` here; now we have `display: block` by default since this is a div; **this adds extra 5px to the height, even though the sum of the height of the children is the same**, and all paddings and margins are the same; any idea on what could cause this?

it's like this:
```html
<div class="container"> <!-- height: 35px (15px+20px) if display: grid, 40px otherwise?? -->
  <div class="child">some content</div> <!-- height: 15px -->
  <div class="child">some content</div> <!-- height: 20px -->
</div>

Reply:
I'll take a look on this now..

Reply:
Plain HTML is not behaving like this, so it is definitely something in our styles. I tried adding just content to the message, and the height was as expected - the same with the child. However when I tried adding the just the timestamp, the height was bigger then the child. I would suggest inspecting the styles for the timestamp (lineHeight, margin.. anything that could affect the height). In the end, we can try setting some additional styling to the div to fit the content...

Reply:
I'll play with these, thanks @mnajdova I'll keep u posted

Reply:
I looked in the redlines and current behavior looks closer to that; the redlines are incomplete anyways so I'll leave it as is